frontend/rolecall/src/app/piece/piece_editor.component.ts

```diff
@@ -26,14 +26,15 @@ type DraggablePosition = {
 type WorkingPiece = Piece & {
   addingPositions: DraggablePosition[];
   originalName: string;
+  isOpen;
```

Comment:
There's no type on `isOpen`, should there be?

Reply:
Yes it is a boolean. Sorry!